.env.uktrade

```diff
@@ -0,0 +1,82 @@
+ANALYTICS_LINK=REPLACE_WITH_GITLAB_SECRET
```

Comment:
I think we'll have an internal chat about allowing third-party .env files in like this, it might set a bad precendent